_component_design/tooltip.md

```diff
@@ -13,28 +13,304 @@ last-modified: 2017-08-17
-    Tooltip does not reactivate on click.
+### Text
+{: .hxSubSectionTitle}
 
--   Tooltip should only be used on icon images, text, and table rows; use when
-    see fit.
+- 14px Roboto Medium for headers
+- 14px Roboto Regular for body copy
 
--   Max width for the tooltip is 440px.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Text specifications for tooltip component."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-spacing-2.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
 
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Tooltip"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Tooltips/tooltip.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Pointer and placement
+{: .hxSubSectionTitle}
+
+- The pointer is 16 px wide at the base.
+- The apex is 8 px from the side of the rectangle. 
+- The pointer should start 12 px from the corner.
+- The side from which the pointer protrudes depends on the location of the trigger on the screen.
+
+	- Always have the tooltip protrude toward the center of the screen
+	- Be sure not to hide important information with the tootip
+	- The popover or tooltip content should be repositioned so that it doesn’t bleed off the screen.
+
+- The depth/width of the remaining side of the tooltip is dynamic based on how large the box needs to be with a max-width of 400px. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Variations of tooltip based on pointer placement."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-pointer-placement.svg){:width="100%"}
 {% endfigure %}
+
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Tooltip specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Tooltips/tooltip-specs.svg){:width="100%"}
+</div>
+
+## Triggers
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Tooltips will only appear after be activated by a trigger. Triggers come in a variety of types.
+
+### Trigger types
+{: .hxSubSectionTitle}
+
+- **Icon:** Use this trigger variation when a recognizable icon on the application informs a user that additional information is available for consumption if needed. (e.g. A ‘?’ for help or ‘$’ for pricing) 
+- **Table text:** This variation is used to reveal additional information within a table. By hiding information behind tooltips in a table we keep the tables easily consumable and only display additional information when a user requires. Do not hide important information behind a tooltip. Note: May apply to both plain text cells and link text cells.
```

Comment:
"By hiding information in tooltips in a table, the tables are easily consumable and only display additional information if required."
You can then probably remove "Do not hide important information behind a tooltip." - just repetitive. 

The 'note' should be called out.

Reply:
Added note as a Sub dot point.